### ISSUE TITLE
User Story FE #6: Accessibility

### ISSUE THREAD

**Issue by Julia-Wagner** [Julia-Wagner/pawfect-match#23]
EPIC: **General UX**

As a **site user**, I expect **the application to be accessible**, so that **everyone can use it**.

**Acceptance Criteria:**
- All parts of the application should be accessible.
- WCAG standards are met.
- WAVE returns no erros.

**Tasks:**
- [x] Implement semantic tags.
- [x] Check WCAG standards.
- [x] Check for WAVE erros.
